@navikt/ds-react/src/copy-to-clipboard/CopyToClipboard.tsx

```diff
@@ -1,39 +1,48 @@
 import { Copy } from "@navikt/ds-icons";
-import copy from "copy-to-clipboard";
 import cl from "classnames";
+import copy from "copy-to-clipboard";
 import React, {
+  ButtonHTMLAttributes,
   forwardRef,
-  HTMLAttributes,
   useEffect,
   useRef,
   useState,
 } from "react";
 import mergeRefs from "react-merge-refs";
-import { Popover, Button } from "..";
+import { BodyShort, Button, ButtonProps, Popover, PopoverProps } from "..";
 
-export interface CopyToClipboardProps
-  extends HTMLAttributes<HTMLButtonElement> {
+export interface CopyToClipboardProps extends Omit<ButtonProps, "children"> {
   /**
    * Button text
    */
   children?: React.ReactNode;
   /**
    * Text to be copied to clipboard
    */
-  text: string;
+  copyText: string;
   /**
    * Description of text, examples: "personnummer", "navn", "epost" etc.
    */
-  label: string;
+  popoverText: string;
   /**
-   * Use a differnet icon to display
-   * @default "Files" from @navikt/ds-icons
+   * Allows extending popover properties like "placement"
    */
-  icon?: React.ReactElement;
+  popoverProps?: Partial<PopoverProps>;
 }
 
 const CopyToClipboard = forwardRef<HTMLButtonElement, CopyToClipboardProps>(
-  ({ children, text, label, icon, className, ...rest }, ref) => {
+  (
+    {
+      children,
+      copyText,
+      popoverText,
+      className,
+      size = "medium",
+      popoverProps,
```

Comment:
Jeg er usikker på om verdien i å eksponere `popoverProps` her er stor nok. Ved å eksponere `popoverProps` skaper vi en sterk binding til `Popover` så det blir vanskelig å f.eks. skrive seg vekk fra den i framtiden. Er det noen spesielle caser du har sett som løses med `popoverProps`?

Reply:
Enig, er ikke nødvendigvis så stor fan heller. Casen var noe Linda nevnte med at de ville eks ha popoveren under istedenfor høyre. Kan evt bare eksponere "placement" her for plassering av popoveren.

Reply:
Ja, da foretrekker jeg å ha en prop som heter "placement"